simple_history/models.py

```diff
@@ -660,6 +660,19 @@ def m2m_changed(self, instance, action, attr, pk_set, reverse, **_):
             # It should be safe to ~ this since the row must exist to modify m2m on it
             self.create_historical_record(instance, "~")
 
+    def _get_through_field_name(self, through_table_fields, model):
+        """
+        Find the name of the field in the through table. This is necessary for the
+        custom through tables where Django conventions don't apply.
+        """
+        foreign_keys = filter(
```

Comment:
add the case where m2m has `through_fields`

https://github.com/django/django/blob/main/django/db/models/fields/reverse_related.py#L383